datachannel_go_test.go

```diff
@@ -345,3 +352,218 @@ func TestDataChannelBufferedAmount(t *testing.T) {
 		assert.True(t, nCbs > 0, "callback should be made at least once")
 	})
 }
+
+func TestEOF(t *testing.T) {
+	log := logging.NewDefaultLoggerFactory().NewLogger("test")
+	label := "test-channel"
+	testData := []byte("this is some test data")
+
+	checkError := func(t *testing.T, err error) {
```

Comment:
Are we trying to eventually remove the `stretchr/testify/assert` dependency? Because if not, you can use `assert.NoError(t, err)` instead of writing your own `checkError` function. It does the same thing.

Reply:
`assert.NoError(t, err)` is good.  will switch to that.

Reply:
Done